apidef/oas/schema/x-tyk-gateway.md

```diff
@@ -1,10 +1,11 @@
+# XTykAPIGateway
 
-## TYK OAS API Definition Object
-
-### **x-tyk-gateway**
+XTykAPIGateway contains custom Tyk API extensions for the OAS definition.
```

Comment:
```suggestion
XTykAPIGateway contains custom Tyk API extensions for the OpenAPI definition.
```

(only suggesting this because OpenAPI Specification Definition seems a bit unwieldy, but can be left as-is if you prefer)